R/pkgs/hospitalization/man/build_hospdeath_geoid_fixedIFR_par.Rd

```diff
@@ -0,0 +1,56 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
We aren't committing the Rd files for packages.  Can you please remove these from the PR

Reply:
Removed